vision/recognition_service.proto

```diff
@@ -173,7 +173,7 @@ message FeatureDescriptorMatch {
   repeated string tags = 1;
 
   // Score for the match from 0 to 1. 
-  // A bigger value means a closer match.
+  // A smaller value means a closer match.
```

Comment:
It's probably better in the long run to adjust the score so we can keep this consistent across algorithms. The closer to 1 the more similar the descriptors.